stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt

```diff
@@ -333,6 +333,7 @@ internal class ChannelLogic(
         setWatcherCount(c.watcherCount)
 
         mutableState._read.value?.lastMessageSeenDate = c.lastMessageAt
+        mutableState._membersCount.value = c.memberCount
```

Comment:
Shouldn't we also do the same in `updateOldMessagesFromChannel`? 🤔

Reply:
Good catch.